webapp-mgmt/cas-management-webapp-configuration/src/main/java/org/apereo/cas/mgmt/configuration/CommaSeparatedStringToThrowablesConverter.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.mgmt.configuration;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.util.ClassUtils;
+import org.springframework.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This is {@link CommaSeparatedStringToThrowablesConverter}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.3.0
+ */
+public class CommaSeparatedStringToThrowablesConverter implements Converter<String, List<Class<? extends Throwable>>> {
```

Comment:
This file need to go away. You should use one that is provided to you by the configuration-api module.